mm2src/coins/Cargo.toml

```diff
@@ -72,7 +72,7 @@ mm2_number = { path = "../mm2_number"}
 mm2_p2p = { path = "../mm2_p2p" }
 mm2_rpc = { path = "../mm2_rpc" }
 mm2_state_machine = { path = "../mm2_state_machine" }
-mocktopus = "0.8.0"
+mocktopus = { version = "0.8.0", optional = true }
```

Comment:
I think the way like it is now is convenient as we may use mocktopus both from tests inside the crate and from other crates, without extra features, and at the same time avoiding building it into releases

Reply:
from tests inside the crate -> dev-deps section
from tests from outside/integration -> `for-test` flag to activate it, this has the downside though as u said will enable all and every extra test features we have, but it's a for-test build anyway so i wouldn't worry much about that and instead prioritize having less complex `cfg`s around the code base.

Reply:
> from tests from outside/integration -> for-test flag to activate it

I think, in this case mocktopus would be built into release what is not recommended for performance reasons.
The optional flag is a way to avoid this.